site/tr.njk

```diff
@@ -12,17 +12,17 @@ lang: tr
     <section id="container-centre" class="column centre flex-1">
 
         <h1 class="page-title">{{ title }}</h1>
-        <h2 class="page-subtitle">COVID-19 Pandemisini Durdurma Hareketi</h2>
+        <h2 class="page-subtitle">COVID-19 Salgınını Durdurma Hareketi</h2>
 
         <div class="content">
 
             <p class="text-xl">
                 Hükümetlerimiz
                 <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
-                salgınını ve <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 pandemisini</a> önlemede başarısız kaldı.
+                salgınını ve <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 salgınını</a> önlemede başarısız kaldı.
                 Yavaş reaksiyonları, insanları yatıştırma politikaları ve ekonomiyi istikrarlı tutma çabaları, milyonları etkileyecek bu hastalığa karşı önlem almalarını engelliyor. Bu dünyanın bir vatandaşı olarak aksiyon almamızın ve COVID-19’a karşı savaşmamızın vakti geldi.
                 <br /><br />
-                Açık dille söylemek gerekirse: <strong>Lanet Olası Evinde Kal!</strong>
+                Açık dille söylemek gerekirse: <strong>Siktiğimin Evinden Dışarı Çıkma!</strong>
```

Comment:
Küfür biraz sert olmuş. Önceki hali yada başka bir alternatif kullanılabilir.

Reply:
Since we don't use the phrase 'lanet olası' much in general, I agree with that stronger expression. Or 'koduğumun' could be a suitable alternative for the purpose of this website.

Reply:
Yeah we do not use but your first alternative is really rude expression. also 'koduğumun' is still rude one.  Maybe we can say "Kıçını even dışarı çıkarma" or  "Kır kıçını otur evinde". These are less rude. What do you think ?

Reply:
olay zaten sert bi tepki koymak bu yuzden kibarlik pesinde kosulmamali, reponun olayini anladigina emin misin? @ozanertuk

Reply:
Küfür bence de hoş olmamış; @ozanerturk'e katılıyorum. @nsa'in gündeme getirdiği üzere "koduğumun" daha idiomatik bir alternatif olabilir ya da belki de "otur oturduğun yerde" gibi bir şey?